### PR TITLE
Make note about servers on a path or operation being ignored clearer

### PR DESCRIPTION
**Pull request by develohpanda**
What it says on the tin. Docs about the overrides that are currently not supported by O2K: https://swagger.io/docs/specification/api-host-and-base-path/#overriding

### CODE REVIEW

File: packages/openapi-2-kong/README.md [hunk 1]
```diff
@@ -164,7 +164,9 @@ The service name is set to the following
 
 ### `$.upstreams`
 
-Upstreams and targets are generated from the `servers` root property (note that `servers` defined on `operation` or `path` objects are ignored).
+Upstreams and targets are generated from the `servers` root property.
+
+> Note: In OpenAPI, `servers` can be overridden on the `operation` or `path` object. These overrides are currently not supported and will be ignored while generating config.
```
Comment (falondarville) [inline packages/openapi-2-kong/README.md]:
```suggestion
> Note: In OpenAPI, `servers` can be overridden on the `operation` or `path` object. These overrides are currently not supported in Insomnia and will be ignored while generating config.
```

Comment (falondarville) [reply packages/openapi-2-kong/README.md]: That clarifies the distinction between, "you can do this as per OpenAPI but over here, it won't work as you expect"

Comment (develohpanda) [reply packages/openapi-2-kong/README.md]:
```suggestion
> Note: In OpenAPI, `servers` can be overridden on the `operation` or `path` object. These overrides are currently not supported by openapi-2-kong and will be ignored while generating config.
```

**Review by falondarville — APPROVED** [review]
LGTM

**Pull request merged**